### PR TITLE
Minor tweaks to the .NET page

### PR DESCRIPTION
**Pull request by justinvp**
Minor tweaks and cleanup to the .NET page.

### CODE REVIEW

File: content/docs/intro/languages/_index.md [hunk 1]
```diff
@@ -17,8 +17,8 @@ capable as the other and supports the entire surface area of resources available
 The following language runtimes are currently supported by Pulumi:
 
 * [Node.js]({{< relref "javascript.md" >}}) - JavaScript, TypeScript, or any other Node.js compatible language
-* [Python 3]({{< relref "python.md" >}}) - Python 3.6 or greater
-* [.NET]({{< relref "dotnet.md" >}}) <span class="badge badge-preview">PREVIEW</span> - C#, F# and Visual Basic 
+* [Python]({{< relref "python.md" >}}) - Python 3.6 or greater
+* [.NET Core]({{< relref "dotnet.md" >}}) <span class="badge badge-preview">PREVIEW</span> - C#, F#, and Visual Basic on .NET Core 3.0 or greater
```
Comment (justinvp) [inline content/docs/intro/languages/_index.md]: Just to match the naming we have in the TOC

File: content/docs/intro/languages/dotnet.md [hunk 2]
```diff
@@ -64,17 +64,14 @@ The fastest way to get up and running is to choose from one of the following Get
     <a class="btn bg-transparent border w-1/2 border-solid border-gray-300 hover:bg-gray-200 justify-center p-4 mr-4" href="{{< relref "../../get-started/gcp" >}}">
         <img class="h-5 ml-auto mr-auto" src="/logos/tech/gcp.svg" alt="Google Cloud">
     </a>
-    <a class="btn bg-transparent border w-1/2 border-solid border-gray-300 hover:bg-gray-200 justify-center p-4" href="{{< relref "../../get-started/kubernetes" >}}">
-        <img class="h-5 ml-auto mr-auto" src="/logos/tech/k8s.svg" alt="Kubernetes">
-    </a>
```
Comment (justinvp) [inline content/docs/intro/languages/dotnet.md]: We don't yet have .NET in the Kubernetes Get Started guide, so removing this temporarily for now.

File: content/docs/intro/languages/dotnet.md [hunk 3]
```diff
@@ -64,17 +64,14 @@ The fastest way to get up and running is to choose from one of the following Get
...
 </div>
 
 > The Getting Started guides currently only demonstrate C#. For F# and Visual Basic, please refer to the
 > documentation below. They will work just fine &mdash; the guides simply aren't ready for them yet.
 
 ## Prerequisites
 
-Before using Pulumi for .NET, you will need to install both Pulumi and .NET Core 3.0 or later. If you follow the Getting Started guides above, they will walk you through doing this.
+Before using Pulumi for .NET, you will need to install both Pulumi and .NET Core SDK 3.0 or later. If you follow the Getting Started guides above, they will walk you through doing this.
```
Comment (justinvp) [inline content/docs/intro/languages/dotnet.md]: It's worth being explicit that the SDK is needed.

File: content/docs/intro/languages/dotnet.md [hunk 4]
```diff
@@ -102,15 +97,12 @@ To deploy your infrastructure run `pulumi up` and the Pulumi engine automaticall
...
 This `csharp` template is cloud agnostic, and you will need to install NuGet packages for the cloud provider of your choice. Additional templates are available:
 
-* `aws-csharp`: a starter AWS project in C#
-* `azure-csharp`: a starter Azure project in C#
-* `gcp-csharp`: a starter GCP project in C#
-{{% /md %}}
-</span>
+* `pulumi new aws-csharp`: creates a starter AWS C# project
+* `pulumi new azure-csharp`: creates a starter Azure C# project
+* `pulumi new gcp-csharp`: creates a starter Google Cloud C# project
```
Comment (justinvp) [inline content/docs/intro/languages/dotnet.md]: Just to make it a little more clear for people skimming the page that these templates are to be used with `pulumi new`.

**Comment by justinvp** [comment]
One thing I've noticed that's kind of annoying... If you click the F# or Visual Basic tab, the scroll position on the page jumps around. It's because of the C# example at the top, which becomes hidden when switching to a different language, due to how the lang chooser currently works. I suspect the jumpiness would be mitigated if we added F# and VB examples at the top of the page as well. I'll try that and add to this PR.

File: content/docs/intro/languages/dotnet.md (hunk 2 above)
Comment (joeduffy) [reply content/docs/intro/languages/dotnet.md]: Ah 😢

Comment (joeduffy) [reply content/docs/intro/languages/dotnet.md]: By the way, this table looked pretty bad on mobile when I looked this morning (wraps funnily) — this reaches the limits of my CSS skills... any idea why?

Comment (justinvp) [reply content/docs/intro/languages/dotnet.md]: > By the way, this table looked pretty bad on mobile when I looked this morning

Ah yes. I'll add a commit here that fixes it.

File: content/docs/intro/languages/dotnet.md (hunk 4 above)
Comment (joeduffy) [reply content/docs/intro/languages/dotnet.md]: LGTM.

**Comment by joeduffy** [comment]
> If you click the F# or Visual Basic tab...

Random aside: "Visual Basic" is pretty lengthy, and especially envisioning a world with TypeScript, JavaScript, Python, C#, F#, _and_ Visual Basic, does it make sense to shorten to VB?

**Comment by justinvp** [comment]
> Random aside: "Visual Basic" is pretty lengthy, and especially envisioning a world with TypeScript, JavaScript, Python, C#, F#, _and_ Visual Basic, does it make sense to shorten to "VB"?

Done.

**Pull request merged**